src/main/features/base.json

```diff
@@ -358,6 +358,10 @@
             "user.mapping":[
                 "org.apache.sling.jcr.contentloader=[sling-jcr-content-loader]"
             ]
+        },
+        "org.apache.sling.jcr.contentloader.internal.ContentReaderWhiteboard": {
+            // SLING-11168 - ensure all four ootb readers are started
+            "contentReader.cardinality.minimum": 4
```

Comment:
I am in favour of a fix which doesn't require additional OSGi configuration (like https://github.com/apache/sling-org-apache-sling-jcr-contentloader/pull/12) in order for other distributions also benefit from it automatically (without having to configure anything explicitly).

Reply:
I still prefer the fix from https://github.com/apache/sling-org-apache-sling-jcr-contentloader/pull/12/files as it requires the 4 relevant content readers (not only 4 arbitrary ones).

Reply:
i do not like the approach with "minimum=4" (wherever it is defined) - this is too subtle and will likely be forgotten if someone adds a 5th content reader implementation in the future - and then it may produce unreliable results again when it may happen that one random of the 5 readers is not ready. it may also be unreliable when there are custom content readers in the system from other bundles.

Reply:
My understanding is that the contentloader fix is still under review, with an alternate proposal at https://github.com/apache/sling-org-apache-sling-jcr-contentloader/pull/13 . 

I would suggest that we merge this fix for Starter 12, revert it after the release, and then update to the latest JCR ContentLoader release once we have a fix.

How does that sound?

Reply:
that would be possible, but is it worth taking those extra-roundtrip? i think we are very close to the final solution, we have even two PRs in place which actually fix the problem. once we have the final solution ready we could release a new contentloader bundle version together with the final sling 12 starter?

Reply:
I haven't followed the conversations closely. If we think are close to a final solution, then we can wait for a couple of days more, sure.